CONTRIBUTING.md

```diff
@@ -21,6 +22,44 @@ git remote add upstream https://github.com/FerretDB/FerretDB.git
 
 ### Setting up the development environment
 
+In the github-actions there are checks that new code complies the imports sorted convention
```

Comment:
"github-actions" is a repository name. You probably mean "CI" there.

Reply:
fixed